docs/administration-guide/02-setting-up-email.md

```diff
@@ -16,32 +22,36 @@ You should see this form:
 
 ![Email Credentials](images/EmailCredentials.png)
 
-**If you use Google Apps:**
+### Google Apps
+
+1. In the **SMTP host** field, enter smtp.gmail.com
+2. Fill in 465 for the **SMTP port** field
+3. For the **SMTP Security** field, enter **SSL**
+4. In the **SMTP username** field, enter your Google Apps email address (e.g. hello@yourdomain.com)
+5. Enter your Google Apps password in the **SMTP password** field
+6. Enter the email address you would like to be used as the sender of system notifications in the **From Address* field.
+
+### Amazon SES
 
-* In the **SMTP host** field, enter smtp.gmail.com
-* Fill in 465 for the **SMTP port** field
-* For the **SMTP Security** field, enter **SSL**
-* In the **SMTP username** field, enter your Google Apps email address (e.g. hello@yourdomain.com)
-* Enter your Google Apps password in the **SMTP password** field
-* Enter the email address you would like to be used as the sender of system notifications in the **From Address* field.
+1. Log on to [https://console.aws.amazon.com/ses](https://console.aws.amazon.com/ses).
+2. Click **SMTP Settings** from the navigation pane.
+3. Select **Create My SMTP Credentials** in the content pane.
+4. Create a user in the **Create User for SMTP** dialog box and then click **Create**.
+5. Next, select **Show User SMTP Credentials** to view the user's SMTP credentials.
+6. Go back to the Metabase Admin Panel form and enter the info there.
 
-**If you use Amazon SES:**
+**Note**
 
-* Log on to [https://console.aws.amazon.com/ses](https://console.aws.amazon.com/ses).
-* Click **SMTP Settings** from the navigation pane.
-* Select **Create My SMTP Credentials** in the content pane.
-* Create a user in the **Create User for SMTP** dialog box and then click **Create**.
-* Next, select **Show User SMTP Credentials** to view the user's SMTP credentials.
-* Go back to the Metabase Admin Panel form and enter the info there.
+Check if [email quotas](https://docs.aws.amazon.com/ses/latest/dg/quotas.html) apply to your Amazon SES server. You may want to manage your email recipients using groups instead.
```

Comment:
@jeff-bruemmer or @paoliniluis -- any suggestions for a link to add under "manage your email recipients using groups"?

Reply:
nope, I think it's fine :)